src/server.h

```diff
@@ -790,6 +790,7 @@ typedef struct dbBackup dbBackup;
 /* Client MULTI/EXEC state */
 typedef struct multiCmd {
     robj **argv;
+    int argv_len;
     int argc;
     struct redisCommand *cmd;
 } multiCmd;
```

Comment:
Do we really need to store argv_len? It's used while growing the allocation, but after queuing a command, I can't see that it's needed anymore. Oops now it got merged while writing this... Well, it doesn't matter.

Reply:
I think it is needed in case the command uses `rewriteClientCommandArgument` see #9598